### DISCUSSION TITLE
Invites takes you tol login

### DISCUSSION THREAD

**Discussion by jesperordrup** [plausible/analytics#1835] [Self-Hosted Support]
When inviting ppl they cannot register. The emailed link takes them to the login screen.

1) I login and invite  ( enter email )
2) email is received 
3) click the link to create 
4) presented with login screen 

It doesn't matter if I invite an admin or viewerr.

Try to login say the account does not exist (which I wouldn't expect either)

**Comment by jesperordrup** [comment]
I haven't configured the server open for registrations because I don't want anybody to create an account and start collecting stats on their own. 

Is this why I can't invite users to access a website stat ?

https://github.com/plausible/analytics/issues/1706

**Comment by jesperordrup** [comment]
So I tried enabling registration to allow invites to create their account ... 

Sure enough it works. But it seems wrong that you have to allow for anyone to create accounts. 

I suggest that you allow for

No registration
Only registration for invited emails
Open registration.

**Reply by ukutaht** [reply]
Good suggestion! Thanks!

**Reply by jesperordrup** [reply]
Maybe reopening this one: https://github.com/plausible/analytics/issues/1706 (it feels like a bug actually)

**Reply by bitwon** [reply]
@jesperordrup @ukutaht Same needs here. We're also running the self-hosted Plausible, but without opening it for everyone, the invite to a website is useless, but we don't want to share with people the admin account. 

Hope we'll see an improvement in the next release(s).

**Comment by HassanAthmani** [comment]
Update the plausible-conf.env file by including `DISABLE_REGISTRATION=true` then run `sudo docker compose down` and finally `sudo docker compose up --detach` .

**Reply by jesperordrup** [reply]
Thank you. But this is not the answer to this question..yes we know you can disable registrations. 

This questions is about invites

**Reply by HassanAthmani** [reply]
Sorry, my  apologise , I saw an option about  [_Shared links_ ](https://plausible.io/docs/shared-links) , where you can share a unique url with a user so that they can view the stats, I hope that will help.
